mflowgen/common/soc-rtl-v2/run.sh

```diff
@@ -5,6 +5,7 @@ CUR_DIR=$(pwd)
 
 # Clone the SoC Repo
 git clone https://github.com/StanfordAHA/AhaM3SoC.git aham3soc
+(cd aham3soc; git checkout phy_pads)
```

Comment:
Instead of doing this, let's just merge the aham3soc pr first.

Reply:
If I merge aham3soc first, then garnet master will break, because the new pads mess up the old (garnet master) routing.
If we merge garnet first (this merge), nothing breaks ever, because the new routes use the new pads, first by way of aham3soc phy_pads branch, and then after the merge, by way of aham3soc master (step 3).
Does this make sense?
Does that make sense?

Reply:
I totally understand, but it will be in a failing state for about 3 seconds. I don't think it's worth doing all of these separate merges.

Reply:
I don't like failing states. But I will go ahead and delete the line and file the aham3soc pull, you/group can decide when/if you want to approve and/or merge that. Also: I did not intend to repeat the last line twice in the previous comment, I think it sent the wrong message :)

Reply:
Wait don't do that. I want to keep this on hold until we resolve the current fullchip LVS issues.

Reply:
We also don't have the updated RDL

Reply:
Don't worry we're not merging this yet. I will update the plan to reflect the extra step
1. get fullchip LVS working and merged
2. get RDL from PHY guys and get it working in this branch
3. merge this branch and new aham3soc to master